storage/tokudb/ha_tokudb_admin.cc

```diff
@@ -43,13 +43,11 @@ class recount_rows_t : public tokudb::background::job_manager_t::job_t {
     virtual ~recount_rows_t();
 
     virtual const char* key();
-
-    virtual void status(
-        char* database,
-        char* table,
-        char* type,
-        char* params,
-        char* status);
+    virtual const char* database();
+    virtual const char* table();
+    virtual const char* type();
+    virtual const char* parameters();
+    virtual const char* status();
```

Comment:
No, specifically not const as they may need to alter derived class state in order to provide their responses. That _could_ be overridden in the derived class but then violates the stated intention of the API if defined as const.